src/data/set/function.lean

```diff
@@ -630,6 +630,12 @@ lemma bij_on.mk (h₁ : maps_to f s t) (h₂ : inj_on f s) (h₃ : surj_on f s t
 lemma bij_on_empty (f : α → β) : bij_on f ∅ ∅ :=
 ⟨maps_to_empty f ∅, inj_on_empty f, surj_on_empty f ∅⟩
 
+lemma bij_on.inter' (h₁ : bij_on f s₁ t₁) (h₂ : maps_to f s₂ t₂)
```

Comment:
I feel like this should have `left` or `right` in it's name somehow, with another variant for the `bij_on` and `maps_to` arguments swapped.

Reply:
`bij_on.inter_maps_to` and `maps_to.inter_bij_on`?